server/account/account.go

```diff
@@ -142,6 +142,15 @@ func (s *Server) CanI(ctx context.Context, r *account.CanIRequest) (*account.Can
 		}
 	}
 
+	// Application resources have a special notation - for simplicity's sake,
+	// if user gives no sub-resource (or specifies simple '*'), we construct
+	// the required notation by setting subresource to '*/*'.
+	if r.Resource == rbacpolicy.ResourceApplications {
```

Comment:
I see that this logic is duplicated from the Can command. The only difference on top of my head is that the can command provides a user and/or policy instead of the current one. I think the code that validates the permissions for these two commands should be abstracted